packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -672,18 +673,19 @@ export class ModularChangeFamily
 			change.change.destroys === undefined,
 			0x89a /* Unexpected destroys in change to invert */,
 		);
+		const { revInfos } = getRevInfoFromTaggedChanges([change]);
 
 		if (hasConflicts(change.change)) {
 			return makeModularChangeset(
```

Comment:
I agree its the current code is ugly and unclear. I think the best way to improve it would be to refator `makeModularChangeset` so that it takes a bag of named properies (most of which would be optional).

Reply:
I will do it in a separate PR. This PR is anyway too big :-). Created AB#15937 to do this.

Reply:
The comments are still here. Unresolving.

Reply:
Not sure what you mean. Do you want me remove the comments for params?

Reply:
Ah, sorry that was not clear. Yes, that's what I had in mind. It's adding noise to this PR and the comments are likely to go stale without anyone noticing (or be wrong from the start as was the case at one point, I haven't checked the rest).

Reply:
I am working on this already. So, we can leave the comments :-). I will send out a PR soon.